content/post/prioritization_meeting.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Prioritizing Prioritization"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["agile"]
+date: 2018-06-15T17:27:09-04:00
+draft: true
+---
+
+The sprint prioritization meeting is an integral aspect of agile philosophy. You may be more familiar with concepts like stand up, sprint planning, back log grooming and retros, but the prioritization meeting often garners less attention I suspect, partially because it is a difficult process to deploy. It involves regularly occurring meetings, thoughtful ticket (or story) descriptions, considerations of each ticket vis-a-vis every other ticket, and to top it off, this all needs to be coordinated among various Data stakeholders. There is a reason why scrum masters are typically referred to as [cat herders](https://www.google.com/search?ei=GVYpW66lCdHX5gLDsbegDg&q=%22scrum+master%22+%26+%22cat+herder%22&oq=%22scrum+master%22+%26+%22cat+herder%22&gs_l=psy-ab.3..33i160k1.5030.9792.0.10255.6.6.0.0.0.0.139.663.1j5.6.0....0...1c.1.64.psy-ab..0.1.139....0.M519ryGOesY).
+
+<!--more-->
+
+Despite those hurdles, I have come to believe that the end of sprint prioritization meeting is, save for sprint planning, the most crucial project management element of a successful sprint. In this post I will discuss some of the benefit of this meeting, including:
```

Comment:
Might be useful to have  a link to a resource that lays out all of the agile meetings and what they're for.